backend/pkg/api/handlers/public.go

```diff
@@ -657,13 +657,18 @@ func (h *HandlerService) PublicGetValidatorDashboardValidators(w http.ResponseWr
 	}
 	q := r.URL.Query()
 	groupId := v.checkGroupId(q.Get("group_id"), allowEmpty)
-	pagingParams := v.checkPagingParams(q)
+	pagingParams := checkPagingParams[types.ValidatorsCursor](&v, q)
 	sort := checkSort[enums.VDBManageValidatorsColumn](&v, q.Get("sort"))
 	if v.hasErrors() {
 		handleErr(w, r, v)
 		return
 	}
-	data, paging, err := h.getDataAccessor(r).GetValidatorDashboardValidators(r.Context(), *dashboardId, groupId, pagingParams.cursor, *sort, pagingParams.search, pagingParams.limit)
+	search := &types.VDBManageValidatorsSearch{DashboardId: *dashboardId}
+	if !checkSearch(search, pagingParams.search) {
+		returnOk(w, r, emptyPagingResponse())
```

Comment:
needs a return statement after this (also for all other `checkSearch` calls)

Reply:
should maybe also discuss with team whether we want to early return, ignore, or return error (if you did and this was the conclusion please lmk)